### ISSUE TITLE
Questions regarding the code

### ISSUE THREAD

**Issue by SreenivasVRao** [ysharma1126/Split_Brain#1]
Hi, thanks a lot for this implementation! I'm not able to understand why you're reshaping the ab_hat tensor from (?,16,16,256) to (?,256, 625).

My loss is also decreasing rather slowly. 
uns_learning_rate_1=1e-2, #from L -> ab_hat_loss
uns_learning_rate_2=1e-2, #from ab-> L_hat_loss
batch_size=512,
test_size=512

After 500 iterations, uns_L_hat loss is 1000~, while uns_ab_hat_loss is 600~.

I'm also not understanding why your quantize method has 17 bins.

Would be really glad if you can answer these questions. Thanks again!
